sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteIT.java

```diff
@@ -68,6 +68,10 @@
 
   /** Pipeline options for this test. */
   public interface SpannerTestPipelineOptions extends TestPipelineOptions {
+    @Description("Project that hosts Spanner instance")
+    @Default.String("")
+    String getInstanceProjectId();
```

Comment:
You didn't add setter for instanceProjectId, is it optional?

Reply:
It was an omission, thanks.